packages/components/descriptions/src/description.vue

```diff
@@ -0,0 +1,114 @@
+<template>
```

Comment:
Please fix all TS error, and remove path from `internal/build/src/type-unsafe-stricter.json`

Reply:
Sorry for not giving you feedback on this PR for a long time.

I tried to fix the TS error, but I'm having difficulties.

Could it be possible to only work on the switch to script-setup syntax except for the ts error in the PR? All of that work has been done.

Reply:
@sxzz Sorry for calling you directly through the mention, but can you review this PR?